charts/datree-admission-webhook/templates/lease.yaml

```diff
@@ -4,8 +4,10 @@ kind: Lease
 metadata:
   name: datree-webhook-server-lease
   namespace: {{ template "datree.namespace" . }}
+  annotations:
+    "helm.sh/hook": pre-install
   {{- with .Values.customAnnotations }}
-  annotations: {{ toYaml . | nindent 4 }}
+  {{ toYaml . | nindent 4 }}
```

Comment:
needed because argocd doesn't delete this resource upon app deletion

Reply:
I didn't understand this line tbh. how does adding this line help argocd deleting this resource upon app deletion?

Reply:
from helm hooks docs:
```
Executes after templates are rendered, but before any resources are created in Kubernetes
```